libkbfs/block_types.go

```diff
@@ -62,6 +62,19 @@ func (cb *CommonBlock) DataVersion() DataVer {
 	return FirstValidDataVer
 }
 
+// NewEmpty implements the Block interface for CommonBlock
+func (cb *CommonBlock) NewEmpty() Block {
+	return NewCommonBlock()
+}
+
+// Set implements the Block interface for CommonBlock
+func (cb *CommonBlock) Set(other Block) {
```

Comment:
This needs to be called by `DirBlock.Set` and `FileBlock.Set` I think (`CommonBlock` is an embedded field in both of those, I don't think it's used as a `Block` on its own).

Though if you reuse `DeepCopy` as suggested elsewhere, you won't need this at all.